src/components/Form/Location/Address.jsx

```diff
@@ -551,7 +551,7 @@ Address.defaultProps = {
   },
   showPostOffice: false,
   streetLabel: i18n.t('address.us.street.label'),
-  postOfficeStreetPlaceholder: i18n.t('address.apoFpo.street.placeholder'),
+  postOfficeStreetPlaceholder: '',
```

Comment:
we can pull this placeholder default out too

Reply:
also removed 👢